### ISSUE TITLE
Broken reporting of tracing data from services

### ISSUE THREAD

**Issue by pavolloffay** [redhat-developer-demos/istio-tutorial#103]
RP https://github.com/redhat-developer-demos/istio-tutorial/pull/83 broke reporting tracing information from services. The tracing information is reported only from proxies. 

As of istio-0.8 `oc create -f install/kubernetes/istio-demo.yaml` does not deploy jaeger services (the most 
important in this context is jaeger-collector). Services in this repo are configured to send tracing data to this service see https://github.com/redhat-developer-demos/istio-tutorial/blob/master/customer/java/springboot/Dockerfile#L4. 

@christian-posta @objectiser

**Comment by pavolloffay** [comment]
As a quick fix I created a service
```---
apiVersion: v1
kind: Service
metadata:
    name: jaeger-collector
    labels:
      app: jaeger
      jaeger-infra: collector-service
spec:
    ports:
    - name: jaeger-collector-http
      port: 14268
      protocol: TCP
      targetPort: 14268
    selector:
      app: jaeger
    type: ClusterIP
```

and added a port to `rs/istio-tracing-`
```
- containerPort: 14268
112           protocol: TCP

```

**Comment by lordofthejars** [comment]
yeah I know that @christian-posta wanted to check to see which could be the best solution to make everything works smoothly.

**Comment by objectiser** [comment]
Istio 1.0 helm chart should make jaeger-query and jaeger-collector included by default.

**Comment by rafabene** [comment]
I think that was fixed with 1.0, right and #105
